scene/animation/animation_node_state_machine.h

```diff
@@ -149,30 +157,46 @@ class AnimationNodeStateMachine : public AnimationRootNode {
 	struct Transition {
 		StringName from;
 		StringName to;
+		StringName local_from() const {
```

Comment:
While good in intention, this will unfortunately incur a _very_ large performance penality as it beats the purpose of using stringnames (which are just a pointer comparison). The local values need to be cached somehow.

Reply:
Adjusted.